docs/source/developers/index.md

```diff
@@ -461,6 +461,15 @@ custom = "custom_package:CustomChatHandler"
 Then, install your package so that Jupyter AI adds custom chat handlers
 to the existing chat handlers.
 
+## Disabling a built-in slash command
+
+You can disable a built-in slash command globally by providing a mostly-empty chat handler with `disabled = True`. For example, to disable the default `ask` chat handler of Jupyter AI, define:
+
+```python
+class AskChatHandler:
+    disabled = True
```

Comment:
With the above suggestion, developers will also have to set the `id` and `routing_type` field.

```suggestion
class AskChatHandler:
    id = "ask"
    routing_type = SlashCommandRoutingType(slash_id="ask")
    disabled = True
```

Reply:
No longer needed.